server/dive_server/serializers/kwcoco.py

```diff
@@ -0,0 +1,228 @@
+"""
+KWCOCO JSON format deserializer
+"""
+import functools
+import json
+from typing import Any, Dict, List, Tuple
+
+from dive_server.serializers import viame
+from dive_utils import strNumericCompare
+from dive_utils.models import Attribute, CocoMetadata, Feature, Track
+
+
+def is_coco_json(coco: Dict[str, Any]):
+    keys = ['categories', 'keypoint_categories', 'images', 'videos', 'annotations']
+    return any(key in coco for key in keys)
+
+
+def is_coco_string(coco: str):
+    data = json.loads(coco)
+    keys = ['categories', 'keypoint_categories', 'images', 'videos', 'annotations']
+    return any(key in data for key in keys)
+
+
+def annotation_info(
+    annotation: dict, meta: CocoMetadata
+) -> Tuple[int, str, int, List[int]]:
+    annotation_id = annotation['id']
+    image_id = annotation['image_id']
+    filename = meta.images[image_id]['file_name']
+
+    # may not exist, fall back to default values
+    trackId = annotation.get('track_id', annotation_id)
```

Comment:
Looking at Kwcoco (https://github.com/Kitware/kwcoco#the-json-spec)  spec `track_id` could be:
```
    'track_id': <int | str | uuid>  # indicates association between annotations across images
```
Do we need to coerce this into an int somehow?

Reply:
In the case where the `track_id` is a string, it will most likely be in the form of a number in quote like "5" which is trivially easy to parse. If the type is UUID, things get bit more complicated. 

There is a python `uuid` module which can convert something like "cef0cbf3-6458-4f13-a418-ee4d7e7505dd" to a 128-bit integer but that number (although technically valid) would be massive so it would need to be reduced in some way. Another approach could be create a dictionary mappings of UUID to standard track numbers.

Reply:
I think I'd opt for something simple where UUIDs are converted to numbers by putting them all in a set, converting to to a list, and using the index of the UUID as the trackID.